bin/conda_utils.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+conda_ws_name="aurora_conda_ws"
+miniconda_install_root="${HOME}/.shadow_miniconda"
+miniconda_install_location="${miniconda_install_root}/miniconda"
+miniconda_installer="${miniconda_install_root}/miniconda_installer.sh"
+miniconda_installer_url="https://repo.anaconda.com/miniconda/Miniconda3-py311_23.5.2-0-Linux-x86_64.sh"
+miniconda_checksum="634d76df5e489c44ade4085552b97bebc786d49245ed1a830022b0b406de5817"
+packages_download_root="${miniconda_install_root}/aurora_host_packages"
+
+# Some molecule tests install to `/home/...` (no user account)
+if [ -z $USER ]; then
+  if [ -z $MY_USERNAME ]; then
+    HOME='/home'
+  fi
+fi
```

Comment:
So this is only necessary for the molecule tests?

Reply:
I've only seen it needed in molecule tests